### PR TITLE
fix: add support for windows gui clients

### LINKED ISSUE #49: I understand this works with GUI's but what about Github's Desktop Client?
I have installed node with the standard installer.

I have husky working in the CLI but Github Desktop Client acts like there are no scripts to run.

### PR DESCRIPTION
**Pull request by nick-woodward**
Closes #49

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -90,6 +90,14 @@ module.exports = {
...
         'command -v nvm >/dev/null 2>&1 && [ -f .nvmrc ] && nvm use'
       ])
+    } else {
+      // Add
+      // Node standard installation path /c/Program Files/nodejs
+      // for GUI apps
+      // https://github.com/typicode/husky/issues/49
+      arr = arr.concat([
+        'export PATH="$PATH:/c/Program Files/nodejs"'
+      ])
```
Comment (iamstarkov) [inline src/index.js]: will it break unix stuff?

**Comment by nick-woodward** [comment]
Hey @typicode can you look at this when you get the chance?

**Comment by typicode** [comment]
Sure, I hope to be able to do it this week-end

File: src/index.js (hunk 1 above)
Comment (typicode) [inline src/index.js]: Not familiar with adding a PATH in Windows, but out of curiosity `C:\Program...` doesn't work?

Comment (nick-woodward) [inline src/index.js]: I thought the same, but I echoed out the path to a file and they both referenced the C drive in that format.

I ended up testing this with the CLI and GitHub Desktop.

**Comment by typicode** [comment]
Thank you 👍

**Pull request merged**